src/Routes/index.js

```diff
@@ -0,0 +1,22 @@
+import React, {useState,useEffect} from 'react';
+import {BrowserRouter as Router, Route, Switch} from 'react-router-dom'
+import Navbar from '../Components/Navbar'
+import Footer from '../Components/Footer'
+import Request from '../request'
+import Home from '../pages/home'
+function Routing() {
+    
+     return (
+       <>
```

Comment:
it would be nice to add suspense and react lazy to the route setup. It would help code split our project.